docs/pages/text.js

```diff
@@ -92,34 +92,62 @@ export default function TextPage({ generatedDocGen }: {| generatedDocGen: DocGen
       />
       <Example
         description={`
-    You can apply \`size\` options to define the size of the text.
+    You can apply \`size\` options to define the size of the text. These font sizes follow those available through our [Design Tokens](/design_tokens#Font-size).
```

Comment:
Do we need to provide any additional guidance here about when to use various sizes, and Text vs Heading?